dom_test.py

```diff
@@ -0,0 +1,200 @@
+import csv
+import difflib
+import os
+import re
+
+from lxml import etree
+
+ignoredAttrib = {'style', 'type'}
+matched21 = {}
+matched12 = {}
+nodes_info = {1: {}, 2: {}}
+chrome_tree = None
+firefox_tree = None
+THRESHOLD_LEVEL = 0.75
+THRESHOLD_GLOBAL = 0.85
+folder = 'data'
+dom_files_chrome = ['_'.join(f.split('_')[:-1]) for f in os.listdir(folder) if 'dom' in f and 'chrome' in f]
+dom_files_firefox = ['_'.join(f.split('_')[:-1]) for f in os.listdir(folder) if 'dom' in f and 'firefox' in f]
+dom_files = [f for f in dom_files_chrome if f in dom_files_firefox]
```

Comment:
`dom_files = list(set(dom_files_chrome) + set(dom_files_firefox))`

Reply:
I think you meant:
`dom_files = list(set(dom_files_chrome) & set(dom_files_firefox))`

Reply:
Yep, the intersection.